archivy/extensions.py

```diff
@@ -1,23 +1,36 @@
+import os
 import sys
 
 import elasticsearch
+from flask import current_app, g
 from tinydb import TinyDB, Query, operations
 
 from archivy.config import Config
 
-DB = TinyDB(Config.APP_PATH + "/db.json")
```

Comment:
Could you just add a newline here before the start of the code? It's for flake8, I had to fix a conflict and had this issue.